### PR TITLE
update dependencies: 20200203

### PR DESCRIPTION
**Pull request by willkg**
Update dependencies. This covers:

* Bump typed-ast from 1.4.0 to 1.4.1 (from PR #1068)
* Bump importlib-metadata from 1.3.0 to 1.5.0 (from PR #1067)
* Bump botocore from 1.13.47 to 1.14.9 (from PR #1066)
* Bump zipp from 0.6.0 to 2.1.0 (from PR #1065)
* Bump python from `d9c4d8a` to `89d7191` (from PR #1064)
* Bump boto3 from 1.10.47 to 1.11.9 (from PR #1063)
* Bump s3transfer from 0.2.1 to 0.3.2 (from PR #1062)
* Bump pytest from 5.3.2 to 5.3.5 (from PR #1061)
* Bump mako from 1.1.0 to 1.1.1 (from PR #1060)
* Add paul-mclendahand configuration

### CODE REVIEW

**Comment by willkg** [comment]
@jwhitlock Does this look ok?

File: setup.cfg [hunk 1]
```diff
@@ -27,3 +27,7 @@ github_user = mozilla
 github_project = ichnaea
 bugzilla_product = Location
 bugzilla_component = General
+
+[tool:paul-mclendahand]
+github_user = mozilla
+github_project = ichnaea
```
Comment (jwhitlock) [inline setup.cfg]: I get a little itchy with the duplicated key-value pairs, but I don't believe there is an existing ``setup.cfg`` standard to hook into, and I'm not excited about wading into ``setuptools`` just yet. Until then, explicit is better than implicit.

**Comment by willkg** [comment]
I looked in both dependabot and the configuration and couldn't find anywhere that limits to 5 prs at a time. Maybe it's a limit set on the "mozilla" account?

I did go into dependabot and bump the Python dependency thing and that created a handful more PRs.

We don't have this problem with projects under "mozilla-services".

**Comment by willkg** [comment]
Thank you!

**Pull request merged**